File: site/static/templates/blank_play_state/java/lwjgl3/src/main/java/{{PACKAGE_PATH}}/lwjgl3/{{GAME}}Lwjgl3Launcher.java
package {{PACKAGE}}.lwjgl3;

import me.stringdotjar.flixelgdx.backend.lwjgl3.FlixelLwjgl3Launcher;
import {{PACKAGE}}.{{GAME}}Game;

/**
 * Desktop entry point.
 *
 * <p>{@code FlixelLwjgl3Launcher} wires libGDX, logging, and window events for you. The {@link StartupHelper} (same package)
 * restarts the JVM on macOS and NVIDIA Linux when needed.
 *
 * <p>If you're new, don't worry about this file too much; focus on the {@code core} folder, as that's where your
 * game's code will live.
 */
public final class {{GAME}}Lwjgl3Launcher {

  public static void main(String[] args) {
    if (StartupHelper.startNewJvmIfRequired()) {
      return;
    }
    FlixelLwjgl3Launcher.launch(new {{GAME}}Game());
  }

  private {{GAME}}Lwjgl3Launcher() {}
}

File: site/static/templates/blank_play_state/kotlin/lwjgl3/src/main/kotlin/{{PACKAGE_PATH}}/lwjgl3/{{GAME}}Lwjgl3Launcher.kt
package {{PACKAGE}}.lwjgl3

import org.flixelgdx.backend.lwjgl3.FlixelLwjgl3Launcher
import {{PACKAGE}}.{{GAME}}Game

/**
 * Desktop entry point.
 *
 * `FlixelLwjgl3Launcher` wires libGDX, logging, and window events for you. The [StartupHelper] (same package)
 * restarts the JVM on macOS and NVIDIA Linux when needed.
 *
 * If you're new, don't worry about this file too much; focus on the `core` folder, as that's where your
 * game's code will live.
 */
fun main() {
  if (StartupHelper.startNewJvmIfRequired()) {
    return
  }
  FlixelLwjgl3Launcher.launch({{GAME}}Game())
}
